### DISCUSSION TITLE
Entity to trigger backup in automation

### DISCUSSION THREAD

**Discussion by czerus** [lavinir/hassio-onedrive-backup#64] [Ideas]
It would be good to have an entity which allows to trigger backup (apart of schedule) in Automation

**Comment by lavinir** [comment]
Home assistant already exposes a service you can call to perform a backup. If you set the addon to monitor all backups they will be backed up to OneDrive as well. (@czerus )

**Reply by czerus** [reply]
aaa. ok. that solves the problem I have. thanks
